holdings_maintenance/pds4/pds4linkshelf.py

```
@@ -0,0 +1,1270 @@
+            # if not pdsdir.isdir or '_support' in pdsdir.abspath:
+            if not pdsdir.isdir:
+                continue
+
+            # Save logs in up to two places
+            logfiles = set([pdsdir.log_path_for_bundle('_links',
+                                                       task=args.task,
+                                                       dir='pdslinkshelf'),
+                            pdsdir.log_path_for_bundle('_links',
+                                                       task=args.task,
+                                                       dir='pdslinkshelf',
+                                                       place='parallel')])
```

Ruff (0.14.5)

Unnecessary list literal (rewrite as a set literal)

Rewrite as a set literal

(C405)